jaba/src/pages/RequestAccount/RequestAccountPage.css

```diff
@@ -0,0 +1,123 @@
+body {
+    background-color: #012146;
+}
+
+.request-page {
+    width: 1228px;
+    height: 888px;
+    flex-shrink: 0;
+    box-sizing: border-box;
+    margin: 47px 142px 47px 142px;
+    background-color: #ffffff;
+    border-radius: 10px;
+    border: 0.5px solid #878787;
+    box-shadow: 0px 4px 64px 0px rgba(0, 0, 0, 0.05);
+}
+
+.request-header {
+    margin: 39px 769.57px 806px 62px;
```

Comment:
Where are numbers coming from?

Reply:
What is the different between request page and request header? They both appear to wrap all the HTML code